q2-structure/proposal-by-xdg.md

```diff
@@ -0,0 +1,220 @@
+# Proposal for Q2: Structure of Governance
+
+## Definitions
+
+Perl governance will consist of two groups:
+
+* "Core Team"
+
+* "Steering Council"
+
+The specific powers and composition of each are described later.
+
+A section on voting mechanics follows the description of the Core Team and
+Steering Council.  Votes are overseen by a "Vote Administrator".  There are
+five types of votes.  Two are ranked-choice:
+
+* "Term Election"
+
+* "Special Election"
+
+The other three are binary, super-majority votes:
+
+* "Membership Change Vote"
+
+* "Amendment Vote"
+
+* "No Confidence Vote".
+
+The term "Perl Rules of Governance" means the final governance document
+written and ratified by the Gov-Begin group.
+
+## The Perl Core Team
+
+The Perl Core Team are a self-managing group of trusted volunteers involved in
+the ongoing development of the Perl language and interpreter.  They are not
+required to be language developers or committers.
+
+### Powers
+
+The Core Team has authority over the rules and composition of Perl governance.
+
+They alone elect the Steering Council and have the power to remove Steering
+Council members.
+
+In concert with the Steering Council, they manage their own membership and
+have the power to modify the Perl Rules of Governance.
+
+The Core Team do not have any authority over parts of the Perl ecosystem
+unrelated to developing and releasing the language itself.
+
+### Membership, Inactivity, and Removal
+
+The initial Core Team members will be specified in the first draft of the Perl
+Rules of Governance.
+
+Core Team members who have stopped contributing are encouraged to declare
+themselves as "inactive". Inactive members do not participate in votes.
+Inactive members may declare themselves active at any time, except when a vote
+has been proposed and is not concluded.  Eligibility to vote will be
+determined by the Vote Administrator.
+
+Core Team Members may also resign their position at any time.
+
+Any Core Team member may nominate someone to be added to the Core Team by
```

Comment:
This feels counter intuitive to "self-managed".  

You explained this in the PR description as

> Several of the Core Team's decision powers are gated through the Steering Council. Because Core Team voting is costly in time and attention of a large group, this is a mild check against frivolous proposals. Council elections and no-confidence votes are not gated in that way

But I wonder if you're solving a problem that doesn't exist yet?

Reply:
Well, Core picks the Steering Council, so in that sense the system is self-managed.  My goal in Council sign-off is a system that delegates and consolidates governing responsibilities with the council.  I.e. the Core delegates proposal screening to the Council.  Maybe there's a better word than "self-managed"?

PEP 13 gives the Council a veto on additions, but it's after the fact; Council alone does removals; and Council plays no role in amendment.  I wanted a single system for all cases and pre-approval seemed the right balance.

Because there is no quorum requirements for Core voting (for reasons I explained elsewhere), I did want to have at least some formal gate as a check and balance.

Reply:
To me, self-managed would mean that no other group is involved with Core's management.  There might be a better word here.  I think this also gets clearer when the role of the steering council is better explained.  (see other conversation we're having in other comments.)

Reply:
I've removed the word "self-managed".